server/execution/api/patient.py

```diff
@@ -51,7 +51,7 @@ def arrive_patient(patient_id: int):
         return "Missing or invalid request parameter detected.", 400
 
 
-@api.get("/patient/refresh")
```

Comment:
Why is this a post? @lukas0820  
It doesn't update any data, does it?  
Maybe we should also adjust the doc comment, as it isn't correct